src/diffusers/loaders/utils.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
Can we remove this file and put everything in the corresponding classes? I think most things here should go into `lora.py`

Reply:
Just kept it because of `AttnProcLayers` as it is used by custom diffusion.

Reply:
Ok